src/main/resources/db/postgresql/V1_16__CREATE_TABLE_SYSTEM_CONFIGURATION.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS system_configuration;
```

Comment:
should this file be included?

Reply:
yes, as discussed it's for flyway consistency when we merge wcs branch back to cs